scripts/setup_review_app.sh

```diff
@@ -3,20 +3,22 @@
 set -o pipefail
 
 function display_result {
-  EXIT_STATUS=$1
+  RESULT=$1
+  EXIT_STATUS=$2
+  ACTION=$3
 
-  if [ $RESULT -ne 0 ]; then
-    echo -e "\033[31m$Setup failed\033[0m"
+  if [[ $RESULT -ne 0 ]]; then
+    echo -e "\033[31m$ACTION failed\033[0m"
     exit $EXIT_STATUS
   else
-    echo -e "\033[32m$Setup successful\033[0m"
+    echo -e "\033[32m$ACTION successful\033[0m"
   fi
 }
 
-psql -d $DATABASE_URL -c "drop schema public cascade;"
+psql -d $DATABASE_URL -c "drop schema public cascade; create schema public;"
 display_result $? 1 "Drop schema as PR may have migrations not yet applied to master"
 
-pg_dump -F custom --no-acl --no-owner -d $DB_FOR_REVIEW_APPS | pg_restore -c -d $DATABASE_URL
+pg_dump -F custom --schema public -d $DB_FOR_REVIEW_APPS | pg_restore --no-owner --no-acl --schema public -d $DATABASE_URL
```

Comment:
I wonder if we should specify `--no-acl -schema public` in the `pull_prod_data` restore command too? That only has `no-owner` at the moment.

`pull_prod_data` also has the `--no-acl --no-owner` on the `pg_dump`, which I found in the Postgres docs at the time.  I think it makes sense to keep it here too - if these things are never dumped then there shouldn't be any chance of restoring them, as they aren't there in the dump to restore.

Reply:
According to the docs, `--no-acl --no-owner` doesn't really do anything non non-text dumps (i.e. streaming dumps to `pg_restore`). I don't know why this has broken recently, but the information about ownership is definitely being sent to `pg_restore` despite having the flags on `pg_dump`.

Reply:
https://www.postgresql.org/docs/9.6/app-pgdump.html

Reply:
OK fair enough, hadn't read the small print on that.